packages/app/src/utils/use-accessibility-options.ts

```diff
@@ -0,0 +1,75 @@
+import { assert } from '@corona-dashboard/common';
+import { isDefined } from 'ts-is-present';
+import { useIntl } from '~/intl';
+
+export interface AccessibilityOptions {
+  key: string;
```

Comment:
you can make this more strict and get rid of type casting on line 26

```suggestion
  key: keyof typeof siteText.accessibility.charts
```

Reply:
this unfortunately does not work, since the `siteText` is not available there

Reply:
ah I see, try importing the type `SiteText` instead ;-)
```tsx
key: SiteText['accessibility']['charts']

Reply:
shoot, Mondays! 😅